gammapy/stats/feldman_cousins.py

```diff
@@ -0,0 +1,436 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Provide a Feldman Cousins algorithm."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+import matplotlib.pylab as plt
```

Comment:
The imports have been fixed and the test work now.

I removed the plotting option and return the points that were plotted as an extra array.